src/components/landing-components/featured.jsx

```diff
@@ -1,57 +1,31 @@
-import featImg1 from "assets/images/categories/bitsat.jpg";
-import featImg2 from "assets/images/categories/danbrown.jpg";
-import featImg3 from "assets/images/categories/selfhelp.png";
-import featImg4 from "assets/images/categories/schoolbooks.jpg";
-import featImg5 from "assets/images/categories/classics.jpg";
-import featImg6 from "assets/images/categories/poetry.jpg";
 import { FeaturedCard } from "components/cards/cards";
+import { useState, useEffect } from "react";
+import axios from "axios";
 
 export default function Featured() {
+  const [featuredCategories, setFeaturedCategories] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      const response = await axios.get("/api/categories");
+      setFeaturedCategories(response.data.categories);
+    })();
+  }, []);
+
   return (
     <>
```

Comment:
remove this , a return state only requires a single closing div element.

Reply:
remove what? it is a single element